test/bats/aws.bats

```diff
@@ -55,17 +54,15 @@ teardown_file() {
     run kubectl --namespace $NAMESPACE apply -f $PROVIDER_YAML  
     assert_success
 
-    cmd="kubectl --namespace $NAMESPACE wait --for=condition=Ready --timeout=60s pod -l app=csi-secrets-store-provider-aws"
-    wait_for_process $WAIT_TIME $SLEEP_TIME "$cmd"
+    kubectl --namespace $NAMESPACE wait --for=condition=Ready --timeout=120s pod -l app=csi-secrets-store-provider-aws
```

Comment:
How to we arrive at timeout of 120s? I see 60s in some places. Is it based on past test runs?

Reply:
The `120s` timeout is only for the provider image pulls because the image size isn't consistent for all providers. For the e2e cluster the image has to be pulled the 1st time and can take time. For the e2e test image, the image is small and the subsequent tests use `IfNotPresent` in the pod spec. So `60s` for the e2e pods is ample time for the provider to fetch content from external secrets store and driver to complete mount.